### PR TITLE
3.1.1 release notes for main branch

### PR DESCRIPTION
**Pull request by JRahnama**

### CODE REVIEW

**Review by lcheunglci — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -234,6 +234,15 @@ This update brings the below changes over the previous release:
 
 ## [Stable release 3.1.0] - 2022-03-30
```
Comment (Kaur-Parminder) [inline CHANGELOG.md:235]: [Stable release 3.1.1] - 2022-08-12

**Comment by Kaur-Parminder** [comment]
@JRahnama latest stable update is also to be added in release-notes/README.md

File: release-notes/3.1/3.1.1.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 3.1.1 released 12 August 2022
+
+This update brings the below changes over the previous preview release:
```
Comment (Kaur-Parminder) [inline release-notes/3.1/3.1.1.md:5]:
```suggestion
This update brings the below changes over the previous release:
```

File: CHANGELOG.md (hunk 1 above)
Comment (Kaur-Parminder) [inline CHANGELOG.md:235]:
```suggestion
## [Stable release 3.1.1] - 2022-08-12
```

**Comment by JRahnama** [comment]
> @JRahnama latest stable update is also to be added in release-notes/README.md
you mean this `The latest stable release is [Microsoft.Data.SqlClient 5.0](5.0).` Our latest release is still 5.0 and this is a hotfix release. I am not sure if that section should be updated. @DavoudEshtehari  what do you think?

**Review by David-Engel — APPROVED** [review]

**Comment by David-Engel** [comment]
> > latest stable update is also to be added in release-notes/README.md
> 
> you mean this `The latest stable release is [Microsoft.Data.SqlClient 5.0](5.0).` Our latest release is still 5.0 and this is a hotfix release. I am not sure if that section should be updated. @DavoudEshtehari Davoud Eshtehari (Simba Technologies Inc) Vendor what do you think?
> 
> I think when we make this PR to 3.1 servicing branch that section needs to be changed.

Agreed. Latest stable in main should remain 5.0.0. We can update it to 3.1.1 in the servicing branch.

**Pull request merged**